Conf/Package/package_and_sign.sh

```
@@ -67,12 +68,26 @@ readonly DMG_PATH="${ARTIFACTS_DIR}/${RELEASE_NAME}.dmg"
+  )
+
+  if [[ "${BN}" == "sleigh" ]]; then
+    defaults write "${PWD}/parent-launch-constraints.plist" team-identifier "${SIGNING_TEAMID}"
+    CODESIGN_OPTS+=(--launch-constraint-parent "${PWD}/parent-launch-constraints.plist")
+  fi
+
   echo "codesigning ${BN}"
-  /usr/bin/codesign --sign "${SIGNING_IDENTITY}" --keychain "${SIGNING_KEYCHAIN}" \
-        --preserve-metadata=entitlements --timestamp --force --generate-entitlement-der \
-        --options library,kill,runtime "${ARTIFACT}"
+  /usr/bin/codesign ${CODESIGN_OPTS[@]} "${ARTIFACT}"
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)